OrbitQt/orbitmainwindow.cpp

```diff
@@ -1242,3 +1236,24 @@ void OrbitMainWindow::SetupGrpcAndProcessManager(std::string grpc_server_address
   app_->SetGrpcChannel(std::move(grpc_channel));
   app_->SetProcessManager(process_manager_.get());
 }
+
+void OrbitMainWindow::OnProcessListUpdated(std::vector<orbit_grpc_protos::ProcessInfo> processes) {
+  const auto is_current_process = [this](const auto& process) {
+    const ProcessData* const target_process = app_->GetTargetProcess();
+    return target_process != nullptr && process.pid() == app_->GetTargetProcess()->pid();
+  };
+  const auto current_process = std::find_if(processes.begin(), processes.end(), is_current_process);
```

Comment:
Another `auto` that's probably not conforming to the style guide

Reply:
I'm not sure if `const std::vector<ProcessInfo>::const_iterator` helps readability here. Actually I believe it rather harms readability. WDYT?

Reply:
Oh sorry, I misread and thought this was supposed to be `ProcessData*`. My bad, ignore my comment.

Reply:
No worries at all. Glad that we agree on this. ;-)

Reply:
and FWIW. `std::vector<ProcessInfo>::const_iterator` might be an alias for `const ProcessInfo*` depending on the implemenation of the standard library. But we can't assume that.